Cargo.toml

```diff
@@ -1,17 +1,28 @@
 [package]
 name = "netcdf"
-version = "0.1.0"
-authors = ["Michael Hiley <mike.hiley@ssec.wisc.edu>"]
+version = "0.3.2"
```

Comment:
I'll use external maintainer eyes (I still don't know this crate well). Whether people even want version bumps to happen in feature PRs depends on the maintainer, no clue here.

But if there's nothing else going no that I don't know, the next version after 0.1 should be 0.2.0 or at least a 0.x.0 version.

Reply:
I agree that 0.2.0 makes the most sense!